src/scalar_4x64_impl.h

```diff
@@ -189,7 +190,8 @@ static int secp256k1_scalar_is_high(const secp256k1_scalar *a) {
 static int secp256k1_scalar_cond_negate(secp256k1_scalar *r, int flag) {
     /* If we are flag = 0, mask = 00...00 and this is a no-op;
      * if we are flag = 1, mask = 11...11 and this is identical to secp256k1_scalar_negate */
-    uint64_t mask = !flag - 1;
+    volatile int vflag = flag;
+    uint64_t mask = 0 - vflag;
```

Comment:
nb: this is intentionally `0 - vflag` instead of `-vflag` to avoid warnings in MSVC, see #1293

Reply:
I've changed my mind. This should not be the concern of this PR.